### DISCUSSION TITLE
How to not use a computer in the WTD?

### DISCUSSION THREAD

**Discussion by SamuA00** [aaronse/servo-display#7] [Q&A] [answered]
Hey it's SamuA00 from the WTD size discussion. I am wondering how to use the WTD without a computer. I have everything ready and done, but I don't like how I have to bring my computer to turn it on all the time. Is there anyway I can change the code to where the WTD is only using the wave mode animation without a computer?

**Comment by aaronse** [comment]
Hello @SamuA00, what Mcu/Controller are you using?  WTD 1.0 uses an Arduino UNO.  For WTD 2.0 am using Raspberry Pi 4 or Zero 2.  Asking because my answer/options will vary depending on what you're using.  Cheers!

**Reply by SamuA00** [reply]
I am using the Arduino UNO for the WTD 1.0

**Reply by aaronse** [reply, marked as answer]
Cool!  Changing _mode =  MODE_WAVE; will play Wave pattern by default.  I just updated code to this (also renamed to .ino to help Arduino IDE users) https://github.com/aaronse/ServoPixelDisplay/blob/main/src/wtd_1/wtd_1.ino hope that helps.  Cheers!

How's your build turning out?

**Reply by SamuA00** [reply]
Thank you so much. Was exactly what I was looking for. My WTD is working great, just have to hang it somewhere. I am planning to modify it to where the pixels move when something is moving by with a camera. Also is it fine if I connect the power supply that holds 10 amps and 5 Volts to my Arduino UNO. Sorry if I am asking too much, I am new in this area/field.

**Reply by aaronse** [reply]
Most welcome @SamuA00 .  Good luck with hanging your build.  Interaction would be cool.  Am not a power expert, am still learning and would appreciate info from others, I have similar questions/concerns.  Currently, am following advice like https://learn.adafruit.com/adafruit-neopixel-uberguide/best-practices and using a Capacitor similar to ones Adafruit spec for neopixels to help smooth/buffer microcontroller's power supply from onrush current.
